jicoco-kotlin/src/main/kotlin/org/jitsi/temp/HelloWorld.kt

```diff
@@ -0,0 +1,10 @@
+package org.jitsi.temp
```

Comment:
Is this intentionally here?

Reply:
it is, i just want to make sure there's actually kotlin code/tests compiling to make sure it doesn't get broken.  the intent would be to remove it as soon as real kotlin code is added.